airbyte-integrations/connectors/source-kafka/src/main/resources/spec.json

```diff
@@ -11,6 +11,50 @@
     "required": ["bootstrap_servers", "subscription", "protocol"],
     "additionalProperties": false,
     "properties": {
+      "MessageFormat": {
+        "title": "MessageFormat",
+        "type": "object",
+        "description": "The serialization used based on this ",
+        "oneOf": [
+          {
+            "title": "JSON",
+            "properties": {
+              "deserialization_type": {
+                "type": "string",
+                "enum": ["JSON"],
+                "default": "JSON"
+              }
+            }
+          },
+          {
+            "title": "AVRO",
+            "properties": {
+              "deserialization_type": {
+                "type": "string",
+                "enum": ["AVRO"],
+                "default": "AVRO"
+              },
+              "deserialization_strategy": {
+                "type": "string",
+                "enum": ["TopicNameStrategy","RecordNameStrategy","TopicRecordNameStrategy"],
+                "default": "TopicNameStrategy"
+              },
+              "schema_registry_url": {
+                "type": "string",
+                "examples": ["http://localhost:8081"]
+              },
+              "schema_registry_username": {
+                "type": "string",
+                "default": ""
```

Comment:
I ran a local unit test to validate how empty strings are treated, as you are not sanitizing input and taking it directly in. This passed the test.

Reply:
empty string would be removed before adding to kafka client hence, it is ok.